api/tests/config.py

```diff
@@ -0,0 +1,11 @@
+REQUEST_URL = 'http://127.0.0.1'
```

Comment:
Please adapt this to the configuration style we've adopted in `model`

Reply:
this `config.py` is only used in pytest for testing.
it will not be used in production